pkg/ibmcsidriver/controller_helper.go

```diff
@@ -282,6 +320,57 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 	return volume, nil
 }
 
+// setSecurityGroupList
+func setSecurityGroupList(volume *provider.Volume, value string) {
+	securityGroupstr := strings.TrimSpace(value)
+	securityGroupList := strings.Split(securityGroupstr, ",")
+	var securityGroups []provider.SecurityGroup
+	for _, securityGroup := range securityGroupList {
+		securityGroups = append(securityGroups, provider.SecurityGroup{ID: securityGroup})
+	}
+	volume.VPCVolume.SecurityGroups = &securityGroups
+}
+
+// setISENIEnabled
+func setISENIEnabled(volume *provider.Volume, key string, value string) error {
+	var err error
+	if value != TrueStr && value != FalseStr {
+		err = fmt.Errorf("'<%v>' is invalid, value of '%s' should be [true|false]", value, key)
+	} else {
+		if value == TrueStr {
+			volume.VPCVolume.AccessControlMode = SecurityGroup
+		} else {
+			volume.VPCVolume.AccessControlMode = VPC
+		}
+	}
+
+	return err
+}
+
+// setPrimaryIPID
+func setPrimaryIPID(volume *provider.Volume, key string, value string) error {
+	var err error
+	if volume.VPCVolume.PrimaryIP == nil {
+		volume.VPCVolume.PrimaryIP = &provider.PrimaryIP{PrimaryIPID: provider.PrimaryIPID{ID: value}}
+	} else {
+		err = fmt.Errorf("invalid option either provide primaryIPID or primaryIPAddress: '%s:<%v>'", key, value)
+	}
+
+	return err
+}
+
+// setPrimaryIPAddress
+func setPrimaryIPAddress(volume *provider.Volume, key string, value string) error {
+	var err error
+	if volume.VPCVolume.PrimaryIP == nil {
```

Comment:
Same comment here, if we are checking `if len(value) != 0` in case, then in what scenario would we hit the if/else condition?